src/cloud/auth.rs

```diff
@@ -0,0 +1,70 @@
+use std::time::Duration;
+
+use async_std::task;
+
+use crate::cloud::client::{cloud_config_file, CloudClient, CloudConfig};
+use crate::cloud::options;
+use crate::options::CloudOptions;
+use crate::portable::local::write_json;
+use crate::print;
+
+const AUTHENTICATION_WAIT_TIME: i32 = 5 * 60;
```

Comment:
Durations are easier to reason about (i.e. not guessing on units)

Also not sure is 5 minutes is enough.